tests/acceptance/test_files/service_and_plugin.bats

```diff
@@ -450,6 +450,16 @@ load "$LIB_BATS_SUPPORT/load.bash"
   assert_equal "$output" "$(cat $TEST_DATA_DIR/expected_plugin_list_table.txt)"
 }
 
+@test "plugin list - output json" {
+  export STEAMPIPE_DISPLAY_WIDTH=100
```

Comment:
We only set this ENV var for our table display tests(to set the terminal width in GitHub action runners). We don't need to add it while testing JSON outputs.
It is necessary [here](https://github.com/turbot/steampipe/pull/3529/files#diff-dfceac5c03645d4bb51e8aa1e7bd637a2c6394e55f057d5c5745550eec83eebcR491), but remove from [here](https://github.com/turbot/steampipe/pull/3529/files#diff-dfceac5c03645d4bb51e8aa1e7bd637a2c6394e55f057d5c5745550eec83eebcR476) and [here](https://github.com/turbot/steampipe/pull/3529/files#diff-dfceac5c03645d4bb51e8aa1e7bd637a2c6394e55f057d5c5745550eec83eebcR503).